### DISCUSSION TITLE
Just had an update (love the new popup examples in styles) but performance has absolutely TANKED!

### DISCUSSION THREAD

**Discussion by Vanteal** [lllyasviel/Fooocus#1822] [General]
I use stability matrix just so I can have all my tools in one condensed spot. Everything, including Fooocus was working perfectly normal a couple of hours ago, then after dinner, I came back and there's a new update. So I update, and now the performance has absolutely TANKED! Not just a little, but by a massive degree. It would take 6-8 seconds to spit out your average image. Now, it takes well over 5-6 MINUTES! Nothing else has changed in my own settings. This is from the update this evening. Is anyone else having issues?

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#fooocus-suddenly-becomes-very-slow-and-i-have-not-changed-anything

**Reply by Vanteal** [reply]
Thanks, but none of those Q&As apply to my situation.

**Comment by juliux32** [comment]
Sometimes I have a similar problem after updates and reinstalling fixes this problem. Don't know why it works, because you are basically downloading the same thing 🙄 but maybe during update, it leaves some files from previous version and conflicts start to happen

**Reply by Vanteal** [reply]
Yeah, that was one of the first things I did was to reinstall it. Sadly, it didn't help. The other image producers work like normal.

**Comment by lllyasviel** [comment]
hi we cannot analyse it without any info. please open an issue if you have full console log
or you can compare previous versions here https://github.com/lllyasviel/Fooocus/discussions/1405
